config.yaml

```diff
@@ -18,3 +18,9 @@ options:
     description: Kratos UI URL
     type: string
     default: "http://127.0.0.1:4455/"
+  identity_schemas:
+    description: A mapping of schema_id to identity schemas.
```

Comment:
Maybe provide an example here in the description and/or in the README of how this mapping should look like.